selftests/.data/test_statuses.py

```diff
@@ -311,20 +310,3 @@ def tearDown(self):
         raise ValueError
         # pylint: disable=W0101
         self.log.info('teardown post')
-
-
-class KillTest(Test):
-
-    def setUp(self):
-        self.log.info('setup pre')
-        self.log.info('setup post')
-
-    def test(self):
-        self.log.info('test pre')
-        os.kill(os.getpid(), 9)
```

Comment:
Maybe I'm missing some important use case, but due to the new architecture, I don't think we should kill the parent avocado process if a test would like to kill itself. Yes, probably we have some rooms for improvements here, but I would suggest to create a new issue for that, since it is not supported today.